src/Symfony/Component/Stopwatch/Stopwatch.php

```diff
@@ -99,10 +99,16 @@ public function stopSection($id)
      * @param string      $name     The event name
      * @param string|null $category The event category
      *
+     * @throws \LogicException When the name is not entered as a string.
+     *
      * @return StopwatchEvent
      */
     public function start($name, $category = null)
     {
+        if (!\is_string($name)) {
```

Comment:
The docblock states that the input expected here is a string. Thus I am against making this change as we don't do that anywhere else.

Reply:
It will / has been update in 5.0 anyway ?